### PR TITLE
Update xcode extension

### PR DESCRIPTION
**Pull request by 3llomi**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Thanks for the update @3llomi 🔥 

@SvenTiigi can you check this too?

File: extensions/xcode/src/components/xcode-simulator-application-list/xcode-simulator-application-list-item.component.tsx [hunk 1]
```diff
@@ -95,6 +95,47 @@ export function XcodeSimulatorApplicationListItem(props: { application: XcodeSim
...
+                          title: "App data deleted",
+                          message: `${props.application.name} app data has been deleted`,
+                        });
+                      } catch (error: any) {
+                        await showToast({
+                          title: "Error Deleting app data",
+                          style: Toast.Style.Failure,
+                          message: error.message,
+                        });
+                      }
```
Comment (SvenTiigi) [inline extensions/xcode/src/components/xcode-simulator-application-list/xcode-simulator-application-list-item.component.tsx:132]: If you like you can make use of the [`operationWithUserFeedback`](https://github.com/raycast/extensions/blob/dd9fea8e998eb55b34bb0ea725ad8e9595e2221b/extensions/xcode/src/shared/operation-with-user-feedback.ts#L21-L33) function to add a loading indicator if the process of deletion take some time.
https://github.com/raycast/extensions/blob/dd9fea8e998eb55b34bb0ea725ad8e9595e2221b/extensions/xcode/src/shared/operation-with-user-feedback.ts#L21-L33

**Review by SvenTiigi — CHANGES_REQUESTED** [review]
LGTM 👍, just one small optional recommendation.

File: extensions/xcode/src/components/xcode-simulator-application-list/xcode-simulator-application-list-item.component.tsx (hunk 1 above)
Comment (3llomi) [reply extensions/xcode/src/components/xcode-simulator-application-list/xcode-simulator-application-list-item.component.tsx]: Updated. Please check.

**Review by SvenTiigi — APPROVED** [review]
LGTM 👍

**Pull request merged**